salt/api.sls

```diff
@@ -11,6 +11,8 @@ salt-api:
     - version: {{ salt_settings.version }}
   {%- endif %}
 {% endif %}
+{% if not salt_settings.api.service.ignore %}    
+{% if salt_settings.api.service.enabled %}
   service.running:
```

Comment:
Why not set the default in default.yml:
```
api:
  service:
    state: running
    enabled: True
```
and then reference it in the state
```
service.{{ salt_settings.api.service.state }}:
  - name: {{ salt_settings_api_service }}
  - enable: {{ salt_settings.api.service.enabled }}
```

I think that's much cleaner then using jinja magic.

Reply:
You are totally right! I will change it.
I'll check also what's up with the salt-lint report..

Reply:
nice work @cr1st1p